docs/tutorials/api/makers.ipynb

```diff
@@ -0,0 +1,343 @@
+    "\n",
+    "\n",
+    "### FoV background\n",
+    "\n",
+    "\n",
+    "If the background energy dependent morphology is well reproduced by the background model\n",
+    "stored in the IRF, it might be that its normalization is incorrect and that some spectral\n",
+    "corrections are necessary. This is made possible thanks to the `~gammapy.makers.FoVBackgroundMaker`.\n",
+    "This technique is recommended in most 3D data reductions. For more details and usage, see [fov_background](https://docs.gammapy.org/dev/makers/fov.html).\n",
+    "\n",
+    "Here we are going to use a `~gammapy.makers.FoVBackgroundMaker` that will rescale the background model to the data exluding the region where a known source is present. For more details on the way to create exlusion masks see the [mask maps](mask_maps.ipynb) notebook.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "circle = CircleSkyRegion(center=geom.center_skydir, radius=0.2 * u.deg)\n",
+    "exclusion_mask = geom.region_mask([circle], inside=False)\n",
+    "\n",
+    "fov_bkg_maker = FoVBackgroundMaker(method=\"scale\", exclusion_mask=exclusion_mask)\n",
+    "dataset = fov_bkg_maker.run(dataset)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Other backgrounds production methods are available as listed below.\n",
+    "\n",
+    "### Ring background\n",
+    "\n",
+    "If the background model does not reproduce well the morphology, a classical approach consists\n",
+    "in applying local corrections by smoothing the data with a ring kernel. This allows to build a set\n",
+    "of OFF counts taking into account the inperfect knowledge of the background. This is implemented\n",
+    "in the `~gammapy.makers.RingBackgroundMaker` which transforms the Dataset in a `MapDatasetOnOff`.\n",
+    "This technique is mostly used for imaging, and should not be applied for 3D modeling and fitting.\n",
+    "\n",
+    "For more details and usage, see [ring_background](https://docs.gammapy.org/dev/makers/ring.html).\n",
+    "\n",
+    "\n",
+    "### Reflected regions background\n",
+    "\n",
+    "In the absence of a solid background model, a classical technique in Cherenkov astronomy for 1D\n",
+    "spectral analysis is to estimate the background in a number of OFF regions. When the background\n",
+    "can be safely estimated as radially symmetric w.r.t. the pointing direction, one can apply the\n",
+    "reflected regions background technique.\n",
+    "This is implemented in the `~gammapy.makers.ReflectedRegionsBackgroundMaker` which transforms a\n",
+    "`SpectrumDataset` in a `SpectrumDatasetOnOff`. This technique is only used for 1D spectral\n",
+    "analysis.\n",
+    "\n",
+    "For more details and usage, see [reflected_background](https://docs.gammapy.org/dev/makers/reflected.html).\n",
+    "\n",
+    "\n",
+    "## Stacking of datasets\n",
```

Comment:
I think we can omit stacking in this notebook and it goes in as independent documentation

Reply:
I removed the subsection but I still think that the text is relevant so I moved it as a note about the data reduction loop.